src/base/filesystemwatcher.cpp

```diff
@@ -42,9 +44,11 @@
 #include "base/logger.h"
 #include "base/utils/fs.h"
 
+using namespace std::chrono_literals;
+
 namespace
 {
-    const int WATCH_INTERVAL = 10000; // 10 sec
+    const std::chrono::duration WATCH_INTERVAL = 10s;
```

Comment:
There is `using namespace std::chrono_literals;` above.

Reply:
IIRC, c++ changes contain cleanups only, am I right?

Reply:
>There is using namespace std::chrono_literals; above.

Sorry, what do you mean?

>IIRC, c++ changes contain cleanups only, am I right?

Yes.

Reply:
>>There is using namespace std::chrono_literals; above.
>
>Sorry, what do you mean?

Nevermind. I misread it.